src/components/calendar/calendar.js

```diff
@@ -101,7 +101,7 @@ class Calendar extends React.Component {
                   id: event.id,
                   summary: event.summary,
                   time: date.format(`HH:mm`),
-                  weekday: date.weekday() - 1,
+                  weekday: date.weekday() === 0 ? 6 : date.weekday() - 1,
```

Comment:
Oh this was an issue on my part - did not realize they were zero indexed. In this case, we should update the weekdays array above and just pass in `date.weekday()`